packages/wouter/src/index.js

```diff
@@ -236,7 +236,7 @@ export const Link = forwardRef((props, ref) => {
         onClick,
         href,
         // `className` can be a function to apply the class if this link is active
-        className: cls?.call ? cls(path === href) : cls,
+        className: cls?.call ? cls(currentPath === (targetPath)) : cls,
```

Comment:
Everything looks good! One small question: do we need extra braces around `targetPath` here?

Reply:
Definitely not :) No idea what I was thinking. Thanks for the catch — fix pushed.